server/entity/travel.go

```diff
@@ -0,0 +1,102 @@
+package entity
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/portal"
+	"github.com/go-gl/mathgl/mgl64"
+	"sync"
+	"time"
+)
+
+// TravelComputer handles the interdimensional travelling of an entity.
+type TravelComputer struct {
+	// Instantaneous is a function that returns true if the entity given can travel instantly.
+	Instantaneous func(world.Entity) bool
```

Comment:
Does an entity instance have to be passed here?

Reply:
No, it doesn't. Changed.